### DISCUSSION TITLE
Newest Overleaf Version

### DISCUSSION THREAD

**Discussion by matteo-tafuro** [fmarotta/kaobook#117] [Q&A]
Hello,

I'm using kaobook to write my thesis (in Overleaf) and I'm encountering issues that, after some research, seem to be already solved (for example \sidecite still gives me overlapping lines). Therefore I was wondering, is the file on [latextemplates.com](latextemplates.com) the latest version? If not, is there I way I can manually import it in Overleaf?

**Comment by matteo-tafuro** [comment]
Nvm, I solved it. I downloaded the repo and replaced the files manually. However, now I have another question: how do I get the citations to:
- be superscripted and without square brackets
- to appear in the margin as: Authors (Year)
As shown in the image below?

<img width="412" alt="image" src="https://user-images.githubusercontent.com/81420777/112630851-778c6580-8e36-11eb-846b-12dbf0247251.png">

**Reply by fmarotta** [reply]
That's great! Could you please open a new discussion for the second question? Just to keep the posts organised. Thank you!
